lib/validate/validate_function.js

```diff
@@ -143,8 +143,8 @@ module.exports = function validateFunction(options) {
             return [new ValidationError(options.key, options.value, '%s stop domain type must match previous stop domain type %s', type, stopKeyType)];
```

Comment:
Do you think we should rename this function to `validateDomainValue` to ensure mistakes like this are more obvious in the future?